src/pages/organize/[orgId]/people/index.tsx

```diff
@@ -18,16 +19,17 @@ const scaffoldOptions = {
 export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
   const { orgId } = ctx.params!;
 
-  const apiClient = new BackendApiClient(ctx.req.headers);
-  const views = await apiClient.get(`/api/orgs/${orgId}/people/views`);
-
-  if (views) {
+  try {
+    const apiClient = new BackendApiClient(ctx.req.headers);
+    // Note: We don't actually care for the returned orgnaization, but we still want to perform
+    // the api request to know if this user may access this particular page.
```

Comment:
This comment refers to "the returned organization", which I believe is a copy-paste error. Personally, I would remove the comment. I usually prefer to avoid comments (because they can become misleading over time if not carefully kept up to date) but I admit this is a grey area.